style.css

```diff
@@ -570,4 +556,129 @@
     height: 14px;
     margin-top: 24px;
   }
+
+  /* Contact Section */
+
+  .contact {
+    max-width: 100vw;
+    min-height: 100vh;
+    border-top-left-radius: 5.5rem;
+    margin-bottom: 50px;
+    padding: 2rem;
+    background: url('images/contact form shapes-mobile@2x.png') no-repeat, #6070ff;
+    background-size: 100vw;
+  }
+
+  .end-text h1 {
+    height: 56px;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-top: 103px;
+    margin-bottom: 20px;
+    margin-left: 24px;
+    margin-right: 24px;
+    text-align: center;
+  }
+
+  .end-text p {
+    height: 102px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    color: #ebebff;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    text-align: center;
+    margin-bottom: 87px;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+
+  #contact-form {
+    margin-top: 87px;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+
+  #name {
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    padding: 15px;
+    text-transform: capitalize;
+  }
+
+  #email {
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    width: 327px;
+    height: 48px;
+    margin-bottom: 20px;
+    padding: 15px;
+    text-transform: capitalize;
+  }
+
+  #msg {
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    width: 327px;
+    height: 170px;
+    padding: 15px;
+    text-transform: capitalize;
+  }
```

Comment:
- [ ] There is an overflow in your contact section that this causing a horizontal scrollbar to surface on your page while in a mobile view. Kindly inspect this and see how you can improve your design to match the Figma design.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/214428215-0b3156f7-2034-4b36-b7e5-22490d15e867.png)|![image](https://user-images.githubusercontent.com/99715640/214428630-41c9c175-e857-4d50-a1af-7702cffb7c65.png)|

- [ ] Another thing you could do to improve this design is to check your alignment to keep them centered as shown in the Figma design ☝️

Reply:
@AbayomiOlaoye hey, I was looking for a solution the whole day, and was not able to fix it, could you maybe give me some hints to fix this issue? I would very much appreciate it! Thank You! 💯